charts/elasticsearch/templates/es-configmap.yaml

```diff
@@ -38,6 +38,12 @@ data:
         enabled: ${HTTP_CORS_ENABLE}
         allow-origin: ${HTTP_CORS_ALLOW_ORIGIN}
 
+    # Setting the ping schedule explictly ensures that
+    # tcp_keep_alive configuration does not cause components
+    # to have their connections terminated.
+    transport:
+      ping_schedule: "5s"
```

Comment:
I have manually deployed this config to prod, and I'm assessing if it will fix the issue. I tested it for a few hours on staging first.

Reply:
I think it's working

![image](https://user-images.githubusercontent.com/7516283/74883071-2dd17180-533e-11ea-8f1e-4f4d3268af6c.png)

Reply:
current state can be checked here if you are an admin on prod cloud https://grafana.gcp0001.us-east4.astronomer.io/d/qfG4Q31mb/platform-overview-draft-2?orgId=1&refresh=10s&fullscreen&panelId=62&from=now-3h&to=now

Reply:
![image](https://user-images.githubusercontent.com/7516283/74945974-95330400-53c6-11ea-8758-f7b984956169.png)